packages/nimble-components/src/anchor-tabs/tests/anchor-tabs.stories.ts

```diff
@@ -0,0 +1,83 @@
+import { html, repeat, when } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { withXD } from 'storybook-addon-xd-designs';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+import '../../all-components';
+
+interface TabsArgs {
+    tabs: TabArgs[];
+    activeId: string;
+    toolbar: string;
+}
+
+interface TabArgs {
+    label: string;
+    id: string;
+    href: string;
+    disabled: boolean;
+}
+
+const overviewText = `Per [W3C](https://w3c.github.io/aria-practices/#tabpanel) - Tabs are a set of layered
+sections of content, known as tab panels, that display one panel of content at a time. Each tab panel has an
+associated tab element, that when activated, displays the panel. The list of tab elements is arranged along
+one edge of the currently displayed panel, most commonly the top edge.`;
+
+const metadata: Meta<TabsArgs> = {
+    title: 'Anchor Tabs',
+    decorators: [withXD],
+    parameters: {
+        docs: {
+            description: {
+                component: overviewText
+            }
+        },
+        design: {
+            artboardUrl:
+                'https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/b2aa2c0c-03b7-4571-8e0d-de88baf0814b/specs'
+        }
+    },
+    // prettier-ignore
+    render: createUserSelectedThemeStory(html`
+        <nimble-anchor-tabs activeid="${x => x.activeId}">
```

Comment:
When I change tabs in the rendered storybook, the first tab is always shown as active. Here's what it looks like after I clicked "Text Field".

![image](https://user-images.githubusercontent.com/10500124/212380815-12fa02f9-8e8a-41ef-b183-ab5d873e3880.png)

Reply:
I guess this is a consequence of `activeId` being a control in the storybook that the user can set any value for. I think it's a good idea to include it to give us a place to document its behavior, but I'd vote not to allow a storybook user to set it and rather include code in the storybook that sets it automatically. 

Then we should also add docs that say it's the app's responsibility to update `activeId` for the current tab.

Reply:
I've added docs and changed the code to set the active id appropriately when clicking or pressing Space/Enter on a tab.